concepts/enums/about.md

```diff
@@ -83,35 +81,25 @@ In Cairo, traits could be defined and implemented on custom enums. This allows f
         Echo: felt252,
         Move: (u128, u128),
     }
-```
-
-The `Message` enum is made up of:
-
-~ `Quit` has no associated value.
-
-~ `Echo` is a single felt252 value.
-
-~ `Move` is a tuple of two u128 values.
 
-```Rust
-trait Processing {
-    fn process(self: Message);
-}
+    trait Processing {
+        fn process(self: Message);
+    }
 
-impl ProcessingImpl of Processing {
-    fn process(self: Message) {
-        match self {
-            Message::Quit => { println!("quitting") },
-            Message::Echo(value) => { println!("echoing {}", value) },
-            Message::Move((x, y)) => { println!("moving from {} to {}", x, y) },
+    impl ProcessingImpl of Processing {
+        fn process(self: Message) {
+            match self {
+                Message::Quit => { println!("quitting") },
+                Message::Echo(value) => { println!("echoing {}", value) },
+                Message::Move((x, y)) => { println!("moving from {} to {}", x, y) },
+            }
         }
     }
-}
 ```
 
-In many situations, enums can come in handy especially when used with the `match` flow as used above in the traits implementation.
+In many situations, enums can come in handy especially when used with the `match` flow as used above in the traits implementation. There is a chapter that explains in detail the `match` flow.
```

Comment:
```suggestion
In many situations, enums can come in handy especially when used with the `match` flow as used above in the traits implementation. Read more about the `match` flow in [The Cairo Book](https://book.cairo-lang.org/ch06-02-the-match-control-flow-construct.html).
```
nit: I thought about this more and I think since we can't link to other chapters in the Exercism track, and students might not be able to start the desired chapter (e.g. did not go through the prerequisite concepts) or the concept might not even be written when the syllabus is launched, it might be better to just point students to The Cairo Book in case they want to read more _now_.